modules/cake.js

```diff
@@ -1,14 +1,13 @@
 // @ts-check
 const Augur = require("augurbot-ts"),
   Discord = require('discord.js'),
-  config = require('../config/config.json'),
   u = require("../utils/utils"),
   Module = new Augur.Module();
 
-function celebrate() {
+function doCelebrate(test) {
```

Comment:
this doesn't need to be renamed, the name is fine as is. also runBirthdays and runCakedays would work

Reply:
celebrate can stay, but production functions should not have test in their name. quite frankly birthdays and cakedays would be fine, but I already renamed those so unless I need to change them again and risk breaking something then they should be good.